client/src/App.svelte

```diff
@@ -51,6 +53,21 @@
     socket.on('users', (newUsers) => {
       users = newUsers
     })
+
+    socket.on('chat', (data) => {
+      const { from, message } = data
+
+      chats = {
```

Comment:
This can also be
```js
chats[from].push({
  content: message,
  owner: from,
})
chats = chats
```

Just another way of doing this that is supported in svelte.